src/utilities/validation.ts

```diff
@@ -80,3 +79,22 @@ export function validateDateWithInterval(
   }
   return { msg: "", err: false };
 }
+
+export function validateConfirmPassword(password: string, confirmPassword: string): validation {
+  if (
+    password.length < CHAR_LIMIT.MIN_PASSWORD ||
+    confirmPassword.length < CHAR_LIMIT.MIN_PASSWORD
+  ) {
+    return {
+      msg: `ช่องนี้ต้องมีตัวอักษรอย่างน้อย ${CHAR_LIMIT.MIN_PASSWORD} ตัว`,
+      err: true,
+    };
+  }
+  if (password !== confirmPassword) {
+    return {
+      msg: "Password และ Confirm Password ต้องเหมือนกัน",
```

Comment:
If for `resetPassword` page the `msg` should be "New Password และ Confirm New Password ต้องเหมือนกัน", but if others think it's okay to use this msg  (msg in current code) in the `resetPassword` page same as in `registration` page, I'm not against it.

Reply:
My fault. I copy the message from wrong page, but if you feel like "New Password และ Confirm New Password ต้องเหมือนกัน" is better, I would change it.

Reply:
Personally, I think adding "New" would be better since it would be the same as place holder text, so the user would not be confused.

Reply:
I guess so 🤔

Reply:
Solved. Pls check 😸